pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2337,3 +2337,93 @@ def test_merge_join_cols_error_reporting_on_and_index(func, kwargs):
     )
     with pytest.raises(MergeError, match=msg):
         getattr(pd, func)(left, right, on="a", **kwargs)
+
+
+@pytest.mark.parametrize(
+    ("input_col", "output_cols"), [("b", ["a", "b"]), ("a", ["a_x", "a_y"])]
```

Comment:
sorry maybe i wasn't clear, can you make a new file called test_merge_cross.py and put these there.

Reply:
Aaah ok, created the file and moved the tests